src/install.js

```diff
@@ -85,10 +85,21 @@ const versionEndpoint = url.parse('https://sapui5.hana.ondemand.com/resources/sa
 const downloadEndpoint = url.parse('https://tools.hana.ondemand.com/additional/');
 
 (async () => {
+  let latestVersionURL
+  // Check if a specific Version of UI5 is needed..
+  // we are in node_modules/sapui5-runtime/src and need to go down to project using our module
+  const packageJson = fs.readJsonSync(path.join(__dirname, '../../../package.json'), { throws: false })
```

Comment:
You can just require the `package.json`. I think this is a bit simpler than your approach. Take a look at my suggestion above.

Reply:
Originally I used require but your eslint config forbid it and the travis build was failing, so I changed it to fs-extra. I can change the .eslintrc to allow `global-require` but then I need to change `no-dynamic-require` as well. If that is ok for you I will do it, I just tried to adhere to your project guidelines :smile: